shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/receiver/ContextFactoryTest.java

```diff
@@ -164,6 +167,32 @@ public void createContextForNotExistingReplicaTable() throws Exception {
     assertThat(replication.getReplicaTable().getDatabaseName()).isEqualTo(DATABASE);
     assertThat(replication.getReplicaTable().getTableName()).isEqualTo(TABLE);
     assertThat(replication.getReplicaTable().getTableLocation()).isEqualTo(REPLICA_DATABASE_LOCATION + "/" + TABLE);
+    assertThat(context.getCircusTrainConfigLocation()).isNull();
+  }
+
+  @Test
+  public void ctConfigLocationGetsAddedToTheContextWhenProvided() {
+    conf.set(CT_CONFIG.key(), "ct-config.yml");
```

Comment:
Adding CT_CONFIG to the Configuration object and testing that it gets set in the Context.